proto/cosmos/protocolpool/v1/types.proto

```diff
@@ -30,4 +30,31 @@ message Budget {
   // For example, if a period is set to 3600, it represents an action that
   // should occur every hour (3600 seconds).
   google.protobuf.Duration period = 8 [(gogoproto.stdduration) = true];
-}
\ No newline at end of file
+}
+
+// ContinuousFund defines the fields of continuous fund proposal.
+message ContinuousFund {
+  // Recipient address of the account receiving funds.
+  string recipient = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // Percentage is the percentage of funds to be allocated from Community pool,
+  // till the `MaxDistributedCapital` is reached or expired.
+  string percentage = 2 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
+    (gogoproto.nullable)   = false
+  ];
+  // MaxDistributedCapital is the threshold amount for distributing funds.
+  // Once MaxDistributedCapital is reached, further fund distribution is halted.
+  cosmos.base.v1beta1.Coin max_distributed_capital = 3;
+  // Optional, if expiry is set, removes the state object when expired.
+  google.protobuf.Timestamp expiry = 4 [(gogoproto.stdtime) = true];
+}
+
+// FundDistribution is used to keep track of distributed amount of recipients.
+message FundDistribution {
+  // Percentage is the percentage of funds allocated from Community pool,
+  // till the `MaxDistributedCapital` is reached or expired.
+  string percentage = 1 [(cosmos_proto.scalar) = "cosmos.Dec", (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec"];
+  // ToClaim is the amount of funds yet to be claimed.
+  cosmos.base.v1beta1.Coin to_claim = 2;
+}
```

Comment:
why do we need another proto here? 

if the denom will be the same then we only need a uint64. 

also we could store this as two separate maps to avoid encoding and decoding on every read